packages/wordlist/src/controllers/user-data-manager.js

```diff
@@ -280,8 +280,9 @@ export default class UserDataManager {
       } else {
         remoteItems = await remoteAdapter.query(data.params)
         if (params.type === 'full') {
-          for (let remoteItem of remoteItems) {
-            await localAdapter.checkAndUpdate(remoteItem, data.params.segment, [remoteItem])
+          for(let remoteItem of remoteItems) {
+            let wodrItem = localAdapter.dbDriver.createFromRemoteData(remoteItem)
```

Comment:
minor point, but maybe fix the typo in the variable? "wordItem" instead of "wodrItem" ?